plugins/modules/database/postgresql/postgresql_set.py

```diff
@@ -314,12 +314,9 @@ def main():
             if value[:-2].isdigit() and unit in value[-2:]:
                 value = value.upper()
 
-    if value and reset:
+    if value is not None and reset:
         module.fail_json(msg="%s: value and reset params are mutually exclusive" % name)
 
-    if not value and not reset:
```

Comment:
Shouldn't this stay and say `if value is None and not reset:`?

Reply:
@felixfontein good spot! I removed this at the beginning and forgot completely struggling with unrelated CI issues:(

Reply:
3-line fix turns out real hell because of CI:)